src/clj/game/core/eid.clj

```diff
@@ -38,6 +39,14 @@
       (swap! state update :effect-completed dissoc (:eid eid))
       results)))
 
+
+(defn effect-sequence
+  [state side eid args func]
```

Comment:
Spacing is a little messed up in this function. I also think the `func` should be first, and the `args` should be last.

Reply:
jk, having seen how it's being used (as a drop-in replacement for `doseq`, this is fine. Function body needs work tho.